pkg/aws/aws_suite_test.go

```diff
@@ -32,15 +32,18 @@ const (
 	testNamespace = "test"
 )
 
+//params is used to pass annotations to the machine spec
```

Comment:
Let's update the comment as

```
params is the variadic parameter of type interface{} for future adaption of the newMachine function. 
The first of the variadic params is reserved for Machine.Spec.NodeTemplateSpec.ObjectMeta.Annotations
``` 
so that it is clear for other developers in futures